src/ext/oblivc/obliv_bits.c

```diff
@@ -1842,6 +681,7 @@ void feedOblivInputs(OblivInputs* spec, size_t count, int party)
     { __obliv_c__##ot rv; \
       OblivInputs spec; \
       setupObliv##tname(&spec,&rv,v); \
+      spec.src_f = v; \
```

Comment:
No worries. These things go slowly. For my part, I seem to have been confused by the existence of a similarly named float-support repo, which I mistook for the one you're attempting to PR (thus the now-deleted comment about failed compilation about which you may have received some sort of notification).

Having switched to the correct repo, I added the following lines near the top of `testFloatOps.oc` (at line 15):

```
int ref = ocBroadcastInt(rand(), 1);
obliv int x2 = feedOblivInt(ref, 2);
int testout;
revealOblivInt(&testout, x2, 0);
printf("DEBUG: %d, %d\n", ref, testout);
```

When I ran it, both parties printed `DEBUG: 16807, 1183010304`

Thoughts?